ci/jenkins/mellanox-ci/jobs/00-functions.yaml

```diff
@@ -0,0 +1,166 @@
+- builder:
+    name: run_ci
+    builders:
+         - shell: |
+            #!/bin/bash -x
```

Comment:
I think script here can be put into a separate sh file as well, like `clean_common.sh`.

Reply:
Done